forms-web-app/src/controllers/your-details.js

```diff
@@ -0,0 +1,31 @@
+const { appealsApi } = require('../lib/appeals-api-wrapper');
+
+const VIEW = {
+  YOUR_DETAILS: 'your-details/index',
+};
+
+exports.getYourDetails = (req, res) => {
+  res.render(VIEW.YOUR_DETAILS, {
+    appeal: req.session.appeal,
+  });
+};
+
+exports.postYourDetails = async (req, res) => {
+  const body = {
+    appellantName: req.body['appellant-name'],
+    appellantEmail: req.body['appellant-email'],
```

Comment:
not validated - outside the scope of this ticket as this page was only added to prove the concept of save / reload / redisplay form data